data/css/blog.scss

```diff
@@ -0,0 +1,111 @@
+@import "project-setup";
+
+@import "base";
+
+.LayoutSidebar {
+  background-color: $background-dark-color;
+  -EknLayout_Sidebar-sidebar-width-small: 252px;
+  -EknLayout_Sidebar-sidebar-width-large: 252px;
+
+  .sidebar {
+    .main-menu {
+      margin: 66px 0px 0px 0px;
+    }
+
+    .CardTitle {
```

Comment:
Seems like you should be able to deduplicate `.CardTitle` with some nested rules here.